src/components/TimeCardPage/SubmitCard.js

```diff
@@ -6,15 +6,27 @@ import Button from 'react-bootstrap/Button';
 
 
 
+
 export default function SubmitCard(props) {
 
     const [submitted, setSubmitted] = useState(false); 
     const [submitDate, setSubmitDate] = useState(null); 
+    const [state, setState] = useState('Rejected'); 
+    const [rejected, setRejected] = useState(true);
+    const [report_type, setReport] = useState('this is rejected'); 
```

Comment:
⛏️ React naming convention goes `[x, setX]`, so this should be `[report, setReport]` or `[reportType, setReportType]`